protos/feast/serving/ServingService.proto

```diff
@@ -19,55 +19,140 @@ syntax = "proto3";
 package feast.serving;
 
 import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
 import "feast/types/Value.proto";
 
 option java_package = "feast.serving";
 option java_outer_classname = "ServingAPIProto";
 option go_package = "github.com/gojek/feast/protos/generated/go/feast/serving";
 
 service ServingService {
-    // Query features from Feast serving storage
-    rpc QueryFeatures (QueryFeaturesRequest) returns (QueryFeaturesResponse) {
-    };
-
     // Retrieve version information about this Feast deployment
-    rpc GetFeastServingVersion (GetFeastServingVersionRequest) returns (GetFeastServingVersionResponse);
-}
+    rpc GetFeastServingVersion (google.protobuf.Empty) returns (GetFeastServingVersionResponse);
+
+    // Get Feast serving storage type (online or offline)
+    rpc GetFeastServingType (google.protobuf.Empty) returns (GetFeastServingTypeResponse);
```

Comment:
So a serving service will only serve 1 serving type?

Reply:
For now yes.